README.md

```diff
@@ -75,36 +75,41 @@ docker run --restart always --name nix-docker -d -p 3022:22 lnl7/nix:ssh
 
 If you have not setup a remote builder before you can follow these steps.
 
-#### Configure ssh
-
+#### Configure SSH
 ```sh
-mkdir -p /etc/nix
-chmod 600 ssh/insecure_rsa
-cp ssh/insecure_rsa /etc/nix/docker_rsa
+sudo mkdir -p /etc/nix
+sudo chmod 600 ssh/insecure_rsa
+sudo cp ssh/insecure_rsa /etc/nix/docker_rsa
```

Comment:
I think this will depend on the installation type, maybe just duplicate this in the 2 sections and keep everything in ~/.ssh for the single user case.